usaspending_api/broker/management/commands/fix_deleted_fabs.py

```diff
@@ -74,9 +84,9 @@ def handle(self, *args, **options):
             if not rows:
                 logger.info('No further rows; finished')
                 return
-            ids = [r[0] for r in rows]
+            delete_ids = [r[0] for r in rows]
             with timer('deleting rows', logger.info):
-                TransactionNormalized.objects.\
-                    filter(assistance_data__afa_generated_unique__in=ids).delete()
+                fabs_cmd.send_deletes_to_elasticsearch(delete_ids)
```

Comment:
Do you mind spending a minute to rename this function to `send_deletes_to_s3` since it is very inaccurate to say ES anything here. This is the only place it is used outside of `fabs_nightly_loader`